### PR TITLE
Adding code link for examples

### PR DESCRIPTION
**Pull request by erickzanardo**
Adds a link on top of the examples so users can easily see the source code of the example.

On this example you will notice that I created a wrapper for the GameWidget. I did that because I messed up on the code link icon on Dashbook, I left it hardcoded to be black, but our background also is black, so it vanished.

For a next version of Dashbook, I will add another icon there, for information about the example, and will also add a property to customize the color of the icon, so I will do a follow up PR here to address that. 

https://user-images.githubusercontent.com/835641/104042826-b99ff680-51b9-11eb-9711-b8e43debe3cc.mov

### CODE REVIEW

File: lib/stories/animations/animations.dart [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 
 void addAnimationStories(Dashbook dashbook) {
   dashbook.storiesOf('Animations')
-    ..add('basic animations', (_) => GameWidget(game: BasicAnimations()))
-    ..add('aseprite', (_) => GameWidget(game: Aseprite()));
+    ..add(
+      'basic animations',
+      (_) => DashbookGameWidget(game: BasicAnimations()),
+      codeLink:
+          'https://github.com/flame-engine/flame_example/blob/master/lib/stories/animations/basic.dart',
```
Comment (wolfenrain) [inline lib/stories/animations/animations.dart]: Maybe we should make the main path `https://github.com/flame-engine/flame_example/blob/master/` into a const for reuseability?

**Review by wolfenrain — APPROVED** [review]
LGTM. But was wondering if we can change the `code` icon to something else? Maybe a Github icon to showcase where that button will bring you too?

But that might be out of the scope of this PR/project because I dont know if `dashbook` currently supports custom icons

**Review by luanpotter — APPROVED** [review]
LGTM if the constant is added

**Comment by erickzanardo** [comment]
Added the const and a method to make it easier

**Pull request merged**